.github/workflows/closes-issue-check.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow is intended for use in repositories that utilize GitHub Projects.
```

Comment:
Would it be more accurate to say that this is intended for use in repositories that utilize GitHub **issues**... not necessarily projects?

Reply:
Yes, great catch. Updated